.github/workflows/build.yml

```
@@ -72,7 +72,13 @@ jobs:
       uses: docker/setup-buildx-action@5146db6c4d81fbfd508899f851bbb3883a96ff9f
     - if: steps.cache.outputs.cache-hit != 'true'
       name: Build Stellar-Core Image
-      run: docker buildx build --platform linux/${{ inputs.arch }} -f docker/Dockerfile.testing -t stellar-core:${{ inputs.arch }} -o type=docker,dest=/tmp/image https://github.com/stellar/stellar-core.git#${{ env.CORE_REPO_BRANCH }} --build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=true --build-arg CONFIGURE_FLAGS='${{ inputs.core_configure_flags }}'
+      run: |
+        docker buildx build --platform linux/${{ inputs.arch }} \
+        -f docker/Dockerfile.testing -t stellar-core:${{ inputs.arch }} \
+        -o type=docker,dest=/tmp/image \
+        https://github.com/stellar/stellar-core.git#${{ env.CORE_REPO_BRANCH }} \
+        --build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=true \
+        --build-arg CONFIGURE_FLAGS='${{ inputs.core_configure_flags }}'
```

Fyi you can do the following so as to avoid the need for `\` which are easy to forget.
```suggestion
      run: >
        docker buildx build --platform linux/${{ inputs.arch }}
        -f docker/Dockerfile.testing -t stellar-core:${{ inputs.arch }}
        -o type=docker,dest=/tmp/image
        https://github.com/stellar/stellar-core.git#${{ env.CORE_REPO_BRANCH }}
        --build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=true
        --build-arg CONFIGURE_FLAGS='${{ inputs.core_configure_flags }}'
```

---

nice, thanks for suggestion, included it.